setuptools/tests/test_build_meta.py

```diff
@@ -380,7 +380,7 @@ def test_setup_requires(self, setup_literal, requirements, use_wheel,
                 setup(
                     name="qux",
                     version="0.0.0",
-                    py_modules=["hello.py"],
+                    py_modules=["hello"],
```

Comment:
Why this change?

Reply:
The test was failing locally without it. All the other samples in this file with `py_modules` don't have the `.py` extension, and the [packaging guide](https://packaging.python.org/guides/distributing-packages-using-setuptools/#py-modules) says it doesn't belong there.

Reply:
Hm.. It's disturbing that the tests are currently passing if they fail for you locally without that change, no?

Reply:
I think they may have been passing on master and only failed when I made the change I was interested in. As this seemed like a simple mistake and nothing else was failing, I confess I didn't try to understand what was happening.